ANSI Encoding with Umlauts leads to compile errors and javadoc errors

**Describe the bug**
When compiling with Java 17 I get errors due to encoding issues, e.g.:

```
[INFO] --- maven-compiler-plugin:3.10.1:compile (default-compile) @ jollyday-jaxb ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 44 source files to E:\EBS\jollyday-0.7.1\jollyday-jaxb\target\classes
[ERROR] /E:/EBS/jollyday-0.7.1/jollyday-jaxb/target/generated-sources/jaxb/de/focus_shift/jaxb/mapping/HolidayType.java:[4,4] unmappable character (0xC4) for encoding UTF-8
[ERROR] /E:/EBS/jollyday-0.7.1/jollyday-jaxb/target/generated-sources/jaxb/de/focus_shift/jaxb/mapping/HolidayType.java:[16,20] unmappable character (0xFC) for encoding UTF-8
```

The problems are the used Umlauts within the Java files.

**To Reproduce**
Steps to reproduce the behavior:
1) Download current release 0.7.1
2) Compile with Java 17 using the documented command

**Expected behavior**
Compile shouldn't fail.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Win19
- Java 17 Coretto

**Additional context**
Similar problems occur when generating the javadocs. The Umlauts also causes errors.
Maybe converting everything to UTF-8 would be a good solution for the future or omit special characters.

--- a/jollyday-jaxb/pom.xml
+++ b/jollyday-jaxb/pom.xml
@@ -118,6 +118,7 @@
               </systemProperties>
               <jvmArguments>
                 <jvmArgument>-Xms32m</jvmArgument>
+				<jvmArgument>-Dfile.encoding=UTF-8</jvmArgument>
               </jvmArguments>
               <arguments>
                 <!-- These are the arguments you would normally

--- a/jollyday-jaxb/pom.xml
+++ b/jollyday-jaxb/pom.xml
@@ -118,7 +118,7 @@
               </systemProperties>
               <jvmArguments>
                 <jvmArgument>-Xms32m</jvmArgument>
-				<jvmArgument>-Dfile.encoding=UTF-8</jvmArgument>
+                <jvmArgument>-Dfile.encoding=UTF-8</jvmArgument>
               </jvmArguments>
               <arguments>
                 <!-- These are the arguments you would normally

